FakeNFT/Presentation/ProfileFlow/NFTCollection/NFTCollectionViewController.swift

```diff
@@ -0,0 +1,150 @@
+    }
+
+    @objc
+    private func sortButtonTapped() {
+        showActionSheet()
+    }
+
+    private func showActionSheet() {
+        let alertController = UIAlertController(title: Constants.actionSheetTitle,
+                                                message: nil,
+                                                preferredStyle: .actionSheet)
+
+        let sortByPrice = UIAlertAction(title: SortingType.byPrice.rawValue, style: .default) { [weak self] _ in
+            self?.viewModel.sort(by: .byPrice)
+        }
+        let sortByRating = UIAlertAction(title: SortingType.byRating.rawValue, style: .default) { [weak self] _ in
+            self?.viewModel.sort(by: .byRating)
+        }
+
+        let sortByName = UIAlertAction(title: SortingType.byName.rawValue, style: .default) { [weak self] _ in
+            self?.viewModel.sort(by: .byName)
+        }
+
+        let cancelAction = UIAlertAction(title: Constants.cancelAction, style: .cancel)
+
+        let actions = [sortByPrice, sortByRating, sortByName, cancelAction]
+        actions.forEach { alertController.addAction($0)}
+        present(alertController, animated: true)
+    }
+
+    private func setupView() {
+        view.backgroundColor = .white
+        view.addSubview(tableView)
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 30),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    private func setupNavBar() {
+        navigationItem.rightBarButtonItem = .init(customView: sortButton)
+        navigationItem.leftBarButtonItem = .init(customView: backButton)
+        title = "Мои NFT"
+    }
+}
+
+// MARK: - UITableViewDataSource
+extension NFTCollectionViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        viewModel.authorsNames.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard
+            let cell = tableView.dequeueReusableCell(withIdentifier: NFTCollectionCell.reuseIdentifier) as? NFTCollectionCell
+        else { return NFTCollectionCell() }
+        let nft = viewModel.nfts[indexPath.row]
+        let authorName = viewModel.authorsNames[indexPath.row]
+
+        let name = nft.name
+        let rating = nft.rating
+        let author = authorName
+        let price = String(nft.price)
+        let id = nft.id
+        let isLiked = idLikesCollection.contains(Int(id) ?? 0)
+        guard let image = nft.images.first else { return NFTCollectionCell() }
+        cell.configCell(name: name, rating: rating, author: author, price: price, image: image, isLiked: isLiked)
```

Comment:
Можно оставить три параметра в методе и присваивать внутри уже соответствующие свойства модели
```suggestion
        cell.configCell(nft: nft, author: author, isLiked: isLiked)
```